README.md

```diff
@@ -93,3 +93,27 @@ SyncStorage.remove('foo')
     console.log(error);
   });
 ```
+
+#### update(key: _string_, value: _Any type_)
```

Comment:
I think it would to be `getAllKeys`, right? :)

Reply:
Yes, I forgot that.